index.js

```diff
@@ -446,7 +446,7 @@ function format (name, fmt) {
 
 function getFormatFunction (name) {
   // lookup format
-  var fmt = morgan[name] || name || morgan.default
+  var fmt = morgan[name] || name || morgan.combined
```

Comment:
I changed this one also thinking that if "default" is considered as deprecated then it shouldn't be used as a default, right ?

But, although I could create a unitary test for this function along, it didn't make much sense. Then I was not able to create a particular test for this change. I just make sure all other tests kept running fine.

But it would be great to get a confirmation about this change.
Thanks !

Reply:
Yea, this change will break people, so you'll have to determine a different way to make the change. The formats are slightly different, which is why one is deprecated and didn't just change the default without a message if that makes sense.